Automattic/sensei
# Update answer feedback display to use new options

Fixes #4337

### Changes proposed in this Pull Request

* When new options are set, show or hide the feedback elements based on them

### Testing instructions

* Based on #4344 for testing
* Complete a quiz as a learner, then make sure it's graded and the learner failed to pass.
* Toggle the new options in the quiz settings, and check that the elements ("Incorrect" label, correct answer, answer feedback note) are shown only when enabled
* Change the learner's grade and make sure everything is displayed when the learner passed, or if 'Pass required' is turned off

### Notes

* Displaying the correct answer requires the 'Indicate incorrect question' too, since it implies the question was answered incorrectly


### Screenshot / Video

![image](https://user-images.githubusercontent.com/176949/137175822-53b6a105-50b6-4b30-87d7-8adf2d179f89.png)
